setup.py

```diff
@@ -0,0 +1,29 @@
+from setuptools import setup
+
+setup(
+    name='flask-ping',
+    url='https://github.com/libero/flask-ping',
+    project_urls={
+        'Code': 'https://github.com/libero/flask-ping',
+        'Issue tracker': 'https://github.com/libero/publisher/issues'
+    },
+    license='MIT',
+    maintainer='eLife sciences publications',
+    description='Blueprint for a /ping endpoint that can be added to python Flask applications',
+    packages=['flask_ping'],
+    python_requires=">= 3.5",
```

Comment:
how do we go about deciding a minimum Python version here (or more in general)? This comes from what Flask supports I imagine? Or from some 3.5 feature?
It could be even higher if e.g. we want to use 3.7 everywhere for Libero Docker containers.

Reply:
Options after looking at the `.travis.yml` test setup:

- leave just this as the Flask minimal requirement
- specify a stricter range like `>= 3.7` because this is all we intend to use

Reply:
I'm happy to leave this as the Flask minimum as it allows further adoption.